user/serializers.py

```diff
@@ -1,24 +1,67 @@
-from django.contrib.auth import get_user_model
+from django.contrib.auth import get_user_model, authenticate
+from django.utils.translation import gettext as _
 from rest_framework import serializers
 
 
```

Comment:
According to the checklist, you should make sure you've added a blank line at the end of your files. In this file, there are two extra blank lines at the end of the file. Please, remove them.